main.go

```diff
@@ -18,5 +18,5 @@ func main() {
 	}
 	db.ConnectDB(fmt.Sprintf("./sqlite_database/%s.db", dbName))
 	routes.SetRoutes(app)
-	log.Fatal(app.Listen(":3000"))
+	log.Fatal(app.Listen(":8080"))
```

Comment:
@johnhorsema do we need this change?

Reply:
Ah we don't need to. I've changed it to suit my dev environment. Let me push a fix.